RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -335,7 +335,17 @@ class PumpOpsSynchronous {
 
         pages: for pageNum in 0..<16 {
             NSLog("Fetching page %d", pageNum)
-            let pageData = try getHistoryPage(pageNum)
+            let pageData: NSData
+
+            do {
+                pageData = try getHistoryPage(pageNum)
+            } catch let error as PumpCommsError {
+                if pageNum > 0 {
```

Comment:
Once we tell it we want page 1, it either didn't respond or responded with a different opcode. (I didn't check, but I should do that tonight)